tests/fixture/test_data_server_test.go

```diff
@@ -15,19 +15,17 @@ import (
 // Check that funded test keys can be served from an http server to
 // ensure at-most-once allocation when tests are executed in parallel.
 func TestAllocatePreFundedKeys(t *testing.T) {
-	require := require.New(t)
-
 	keys := make([]*secp256k1.PrivateKey, 5)
 	for i := range keys {
 		key, err := secp256k1.NewPrivateKey()
-		require.NoError(err)
+		require.NoError(t, err)
```

Comment:
why? Is there a linter complaining about this?

Reply:
When we use require, we should use it with the most scoped `t` variable. Further down this test, we do `require := require.New(t)` in the sub-test. The linter was complaining that the sub-test `t *testing.T` was unused.